server/utils.go

```diff
@@ -24,3 +26,60 @@ func getGPRCVersion() (string, error) {
 	}
 	return out, nil
 }
+
+func setResourcesCPU(limits, requests, defaultCores float64, g generate.Generator) error {
+	if requests > limits {
+		return fmt.Errorf("CPU.Requests should not be greater than CPU.Limits")
+	}
+
+	cores := defaultCores
```

Comment:
Let's add a reference to the kubernetes document on which our calculations are based on.